### ISSUE TITLE
Download issue with zip file

### ISSUE THREAD

**Issue by fliermate** [jlaasonen/hexfile#2]
Hi there, I am interested to test hexfile.exe, so I try to download [hexfile-v1.1.0-win32.zip](https://github.com/jlaasonen/hexfile/releases/download/v1.1.0/hexfile-v1.1.0-win32.zip) , but on my Windows 11 x64 (Version 10.0.22000.675) with Windows Defender turned on, it reported "Failed - Virus detected" after I insisted to proceed.

![download](https://pictr.com/images/2022/06/13/BOfkgj.png)

`hexfile-v1.1.0-win64.zip`, fortunately, is OK.

**Comment by jlaasonen** [comment]
I have uploaded a new version of [hexfile-v1.1.0-win32.zip](https://github.com/jlaasonen/hexfile/releases/download/v1.1.0/hexfile-v1.1.0-win32.zip). It is built on a different computer and according to Windows Defender it should be clean.

**Comment by fliermate** [comment]
No worries. Users can download FBC and compile on their own.
I managed to use FBC32 and FBC64 to compile `hexfile.bas` successfully without triggering AV.

```
         49,664 hexfile.exe (Compiled with FBC32)
         55,808 hexfile.exe (Compiled with FBC64)
```

**Comment by jlaasonen** [comment]
I tested a bit with [VirusTotal](https://www.virustotal.com) and the false positives appear to be somewhat dependent on the used compiler flags. I was not able to make all the scanners pass, but using -e made Microsoft's scanner pass.

-e should be used anyways as there is currently no error handling in the code and it makes the program exit instead of displaying junk. I updated the compile command in the readme and should swit
